.github/workflows/python-package-conda.yml

```diff
@@ -20,28 +20,41 @@ jobs:
       run: |
         # $CONDA is an environment variable pointing to the root of the miniconda directory
         echo $CONDA/bin >> $GITHUB_PATH
+        echo "source $CONDA/etc/profile.d/conda.sh" >> ~/.bashrc
+
     - name: Install dependencies
       run: |
-        conda env update --file environment.yml --name base
+        bash --login
```

Comment:
I think this needs to be `shell: bash --login {0}` as an option in the step dictionary, not the first line in `run`.

Better would be `shell: bash -o pipefail -o errexit -o nounset --login {0}` to make sure it fails as soon as a command fails and which is basically the default with the addition of `--login`, see:

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell